pkgs/applications/audio/meterbridge/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, pkgconfig, SDL, SDL_image, libjack2
+}:
+
+stdenv.mkDerivation rec {
+  version = "0.9.2";
+  name = "meterbridge-${version}";
+
+  src = fetchurl {
+    url = "http://plugin.org.uk/meterbridge/${name}.tar.gz";
+    sha256 = "0jb6g3kbfyr5yf8mvblnciva2bmc01ijpr51m21r27rqmgi8gj5k";
+  };
+
+  patches = [ ./buf_rect.patch ];
+
+  buildInputs =
+    [ pkgconfig SDL SDL_image libjack2
+    ];
+
+  meta = with stdenv.lib; {
+    description = "meterbridge for Jack Audio Connection Kit";
```

Comment:
The description should not contain the package name. See the nixpkgs manual for further conventions.